src/base/addtorrentmanager.h

```diff
@@ -33,22 +33,23 @@
 
 #include <QHash>
 #include <QObject>
+#include <QString>
```

Comment:
Maybe forward declare it instead?

Reply:
It is used as the key type for `QHash<>` variables so it cannot be forward declared, or am I mistaken?

Reply:
> It is used as the key type for `QHash<>` variables so it cannot be forward declared, or am I mistaken?

Incomplete types are allowed to be used as template arguments.
Besides, you might have noticed that it had no problems before this change.

Reply:
>Incomplete types are allowed to be used as template arguments.

https://godbolt.org/z/hs4jd7MeW
Seems you are right. Done.

>Besides, you might have noticed that it had no problems before this change.

I assume other headers could have included QString header.